src/features/devices/models/device-additions.ts

```diff
@@ -113,36 +204,50 @@ export const addToModel = (abstractSql: AbstractSqlModel) => {
 			[
 				'When',
 				[
-					'Exists',
+					'And',
 					[
-						'SelectQuery',
-						['Select', []],
-						['From', ['Table', 'image install']],
+						'In',
+						['ReferencedField', 'device', 'api heartbeat state'],
+						['EmbeddedText', 'online'],
+						['EmbeddedText', 'timeout'],
```

Comment:
Should we add the same check in the `overall_progress` (and add a v6 translation)?
I kind of can see the use case of showing a progress bar in a reduced-functionality device... so just checking whether this was discussed. 

We would effectively just need to change
```
			[
				'When',
				[
						'NotIn',
						['ReferencedField', 'device', 'api heartbeat state'],
						['EmbeddedText', 'online'],
						['EmbeddedText', 'timeout'],
				],
				// Otherwise if the device is offline but has previously been online we return no info
				['Null'],
			],
```
to
```
			[
				'When',
				isOverallOffline,
				// Otherwise if the device is offline but has previously been online we return no info
				['Null'],
			],
```

Not a blocker though, since we can change it later.